.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,50 @@
+name: Build main branch documentation website
+
+on:
+  push:
+    branches: [main]
```

Comment:
This repo uses `master` instead of `main`

Reply:
Thanks